camel/types/enums.py

```diff
@@ -24,12 +24,11 @@ class RoleType(Enum):
 
 
 class ModelType(Enum):
-    GPT_3_5_TURBO = "gpt-3.5-turbo-1106"
-    GPT_3_5_TURBO_16K = "gpt-3.5-turbo-1106"
+    GPT_3_5_TURBO = "gpt-3.5-turbo"
     GPT_4 = "gpt-4"
     GPT_4_32K = "gpt-4-32k"
-    GPT_4_TURBO = "gpt-4-1106-preview"
-    GPT_4_TURBO_VISION = "gpt-4-vision-preview"
+    GPT_4_TURBO = "gpt-4-turbo"
+    GPT_4_TURBO_VISION = "gpt-4-turbo"
```

Comment:
How about removing this "GPT_4_TURBO_VISION" since it does not match any exact model name? All GPT4 turbo except the first preview version are with vision ability.

Reply:
I agree with @ocss884 , since only version set will make gpt-4-turbo different, we can remove the duplicated ModelType

Reply:
Let's keep this for now and we may remove it after fixing: https://github.com/camel-ai/camel/pull/473#issuecomment-2063584662.